src/main/java/com/questhelper/helpers/quests/onesmallfavour/OneSmallFavour.java

```diff
@@ -524,7 +524,7 @@ public void setupSteps()
 		talkToYanni = new NpcStep(this, NpcID.YANNI_SALIKA, new WorldPoint(2836, 2983, 0), "Talk to Yanni Salika in Shilo Village. CKR fairy ring or take cart from Brimhaven.");
 		talkToYanni.addDialogStep("Yes.");
 		talkToYanni.addDialogSteps("Is there anything else interesting to do around here?", "Ok, see you in a tick!");
-		talkToJungleForester = new NpcStep(this, NpcID.JUNGLE_FORESTER, new WorldPoint(2861, 2942, 0), "Talk to a Jungle Forester south of Shilo Village.", bluntAxe);
+		talkToJungleForester = new NpcStep(this, NpcID.JUNGLE_FORESTER_3955, new WorldPoint(2861, 2942, 0), "Talk to a Jungle Forester south of Shilo Village.", bluntAxe);
```

Comment:
Since both NPCs are fine to talk to, you could keep `NpcID.JUNGLE_FORESTER` as an alternate NPC ID and highlight them all. The text already suggests this kind of. I don't remember how many of them there are so might look messy - up to you!

```suggestion
		talkToJungleForester = new NpcStep(this, new int[]{NpcID.JUNGLE_FORESTER_3955, NpcID.JUNGLE_FORESTER}, new WorldPoint(2861, 2942, 0), "Talk to a Jungle Forester south of Shilo Village.", true, bluntAxe);
```

Reply:
There's only 2 Forester steps, and only 2 Forester NPC IDs, so I've gone ahead and done this to both steps - thank you